src/output.rs

```diff
@@ -26,7 +26,8 @@ impl OutputType {
 
     /// Try to launch the pager. Fall back to stdout in case of errors.
     fn try_pager(quit_if_one_screen: bool, pager_from_config: Option<&str>) -> Result<Self> {
-        let pager_from_env = env::var("BAT_PAGER").or_else(|_| env::var("PAGER"));
+        let pager_from_env =
+            env::var("BAT_PAGER").or_else(|_| env::var("PAGER").map(add_default_flags_to_less));
```

Comment:
The whole point of this was to have a different handling of `PAGER` and `BAT_PAGER`.  We do not want to modify anything for `BAT_PAGER`.